EpubExtractor/Classes/Epub.swift

```diff
@@ -109,5 +109,5 @@ public struct Epub {
         return self.epubContentParser.chapters
     }
     
-    let epubContentParser: EPubContentParser
+    public let epubContentParser: EpubContentParser
```

Comment:
Why do you need this to be public?

Reply:
Hello! :) 

To read contents (all book as a String) we need to access EpubContentParser 
in EpubContentParser I have placed all logic about parsing text as plain String 
so to access book as a String we need to get variable epubContentParser 

some PR more I will think about different solution, place it in, for example, Chapter, this will be more logical, so than we will able to remove this public

Reply:
Would be possible to make a public getter on the Epub for that?
Something like:

```swift
public var allBook: String {
  return epubContentParser.allBook
}
```

Reply:
Of course, I think about that when it was in the development, next commit will repair that

Reply:
currently removed spineitems from contentParser - because this is already in the Book, 
added methods to get all text - in Epub